plugins/techdocs-node/src/stages/publish/awsS3.ts

```diff
@@ -113,11 +131,10 @@ export class AwsS3Publish implements PublisherBase {
     const credentialsConfig = config.getOptionalConfig(
       'techdocs.publisher.awsS3.credentials',
     );
-    const credentials = AwsS3Publish.buildCredentials(credentialsConfig);
-
-    // AWS Region is an optional config. If missing, default AWS env variable AWS_REGION
-    // or AWS shared credentials file at ~/.aws/credentials will be used.
-    const region = config.getOptionalString('techdocs.publisher.awsS3.region');
+    const credentials = AwsS3Publish.buildCredentials(
+      credentialsConfig,
+      region,
```

Comment:
So the credentials now depend on the region, could this lead to surprises for users? My aws knowledge is limited, but if there's something users should be aware of after this upgrade we can add it to the changeset.

Reply:
Add a note to the changeset